iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/TapChangerAttributeTest.java

```diff
@@ -40,6 +41,79 @@ void testTapChangerAttributeName() {
             ((AbstractTapChanger) twt3.getLeg3().getRatioTapChanger()).getTapChangerAttribute());
     }
 
+    @Test
+    void testTapChangerStepsReplacement() {
```

Comment:
I think it would be interesting to add the following tests:
 - replacing valid steps by invalid invalid (`TapChangerStepImpl.validate()` should throw an error);
 - replacing valid steps by no steps.

Reply:
I did for empty steps.
How can I do for the `validate` ? I can only add `double`.
https://github.com/powsybl/powsybl-core/blob/0cd30e5ff8cc75d992fabbc37c5d333c5a19e516/iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TapChangerStepAdder.java#L20-L28

Reply:
The `validate` method fails if one of these attributes is set to `Double.NaN`.
If you use `setR(Double.NaN)` for instance, the replacement should fail.

Reply:
Ah yes or I can just let it empty.... done